README.tpl.md

```diff
@@ -39,38 +39,186 @@ necessary for fetching and updating the packages.
   `nix/sources.nix` file that returns the sources as a Nix object.
 * [Show](#show): shows the packages' information.
 
+The next two sections cover [common use cases](#use-cases) and [full command
```

Comment:
what is the difference between the template and the normal readme?

Reply:
The normal readme is generated from the template. It inlines the output of `niv <cmd> --help`.